### PR TITLE
Add resources tests

### PR DESCRIPTION
**Pull request by itexpert120**
Resolves https://github.com/NEARBuilders/gateway/issues/411

### CODE REVIEW

File: playwright-tests/tests/resources.spec.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  test("Should navigate to default page", async ({ page }) => {
+    const guideButton = await page.getByRole("button", {
+      name: "Guide",
+    });
+    await guideButton.click();
+    const h1 = await page.getByRole("heading", { name: "Resources" });
+    await expect(h1).toBeVisible();
+  });
+
+  test("Should navigate to Working with VM page", async ({ page }) => {
```
Comment (Megha-Dev-19) [inline playwright-tests/tests/resources.spec.js:18]:
```suggestion
  test("Should navigate to 'Working with VM page'", async ({ page }) => {
```

File: playwright-tests/tests/resources.spec.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}?page=resources`);
+  });
+
+  test("Should navigate to default page", async ({ page }) => {
+    const guideButton = await page.getByRole("button", {
+      name: "Guide",
+    });
+    await guideButton.click();
+    const h1 = await page.getByRole("heading", { name: "Resources" });
```
Comment (Megha-Dev-19) [inline playwright-tests/tests/resources.spec.js:14]:
```suggestion
    const pageHeading = await page.getByRole("heading", { name: "Resources" });
```

File: playwright-tests/tests/resources.spec.js (hunk 1 above)
Comment (Megha-Dev-19) [reply playwright-tests/tests/resources.spec.js:18]: let's wrap page headers inside '' to differentiate it

File: playwright-tests/tests/resources.spec.js (hunk 2 above)
Comment (Megha-Dev-19) [reply playwright-tests/tests/resources.spec.js:14]: let's rename it everywhere else, h1 doesn't really make sense

**Review by Megha-Dev-19 — APPROVED** [review]
Left few comments
Pre approving

**Comment by itexpert120** [comment]
Resolved issues @Megha-Dev-19

**Pull request merged**